apps/dashboard/src/app/(main)/(protected)/projects/[projectId]/auth-methods/page-client.tsx

```diff
@@ -332,6 +332,13 @@ export default function PageClient() {
           }}
           hint="Existing users can still sign in when sign-up is disabled. You can always create new accounts manually via the dashboard."
         />
+        <SettingSelect
+          label="Sign-up mode when logging in with same email on multiple providers"
+        >
+          <SelectItem value="block">Block</SelectItem>
+          <SelectItem value="allow">Allow</SelectItem>
+          <SelectItem value="link">Link</SelectItem>
+        </SettingSelect>
```

Comment:
This should explicitly mention OAuht, and have a description for each of the selections

Reply:
Slightly more explicit now